### ISSUE TITLE
New contributors & reviewers welcomed

### ISSUE THREAD

**Issue by robbat2** [radvd-project/radvd#140]
(sticky issue)

Welcome! The `radvd` project would like to have more regular contributors and reviewers present to encourage the long-term health of the project. This includes all persons who package `radvd` on various distributions.

Current maintainers:
- Robin H. Johnson @robbat2  - past Gentoo maintainer for radvd
- @Neustradamus 

Open invitations:
- @prometheanfire - Gentoo maintainer
- @stappersg - Debian maintainer

The development model of `radvd` will ideally continue in such a way that EVERY change is reviewed by multiple people, regardless of authorship, leadership in the `radvd-project` organization should NOT exempt anybody from having their code reviewed. Caveats apply if private reviews are needed for potential security issues in future.

**Comment by stappersg** [comment]
Yes, you are invented to express  _radvd project has my support_.
No demands made, you chip in what you want to chip in.

> Feel welcome

**Comment by XVilka** [comment]
Have you moved to the [LiberaChat](https://libera.chat/) from the FreeNode? Please update the top message then. Feel free to remove my message afterward to avoid noise. Also, the site https://radvd.litech.org/ should be updated.

**Comment by robbat2** [comment]
@XVilka  done

**Comment by stappersg** [comment]
(Update written after the release of version 2.20)


Any time is a good time to join.

There is no reason to remain in doubt about when would be a good moment to request _"formal review privilege"_.
